environments/prod.yml

```diff
@@ -13,3 +13,10 @@ prometheusRules:
     annotations:
       description: "High number of bad responses coming from selected-tests service"
       summary: "Selected tests service giving high number of bad responses over the last hour (current number: {{ $value }}). More information can be seen here: https://grafana.corp.mongodb.com/d/zenZa6oWk/selected-tests-service?orgId=4"
+resources: # Burstable
+  requests: # guaranteed amount of resources
+    cpu: 500m
+    memory: 6Gi
+  limits: # maximum allowed resources
+    cpu: 2
+    memory: 12Gi
```

Comment:
This seems like a lot of CPU and memory. Looking over the [last 7 days](https://grafana.corp.mongodb.com/d/zenZa6oWk/selected-tests-service?orgId=4&from=now-7d&to=now) CPU has been below 0.007 and memory below 0.1 Gi. Should we perhaps do something closer to that in order to make these pods easier to schedule?

Reply:
At the moment, we have been running the reseed the database update in this container. So it would need to be able to burst up to what ever the limits are for that scenario.

The normal web service should be use very little resources.

We should probably not seed the database from this container.